### PR TITLE
style: login page input border

### LINKED ISSUE #10: List All Created Events
Create a page or section where users can view a list of all events created on the platform.

- [ ]  Design the event listing page.
- [ ]  Implement event retrieval and display.

### PR DESCRIPTION
**Pull request by lutfinoorm**
Hey Mas @indrazm and Mba @sarasseres 
This is what I've done in :
- I've edited some style in login page, 
- add page "events"
- style page "events"
- move layout.js in app to the (auth) and create another layout.js for (events)
- delete page.js that is not used. 

hope you can review it.

### CODE REVIEW

File: src/app/(event)/events/page.js
Comment (indrazm) [inline src/app/(event)/events/page.js:1]: Remove div at 5 and 7. it's redundant

File: src/components/Event/component/event.Card.jsx
Comment (indrazm) [inline src/components/Event/component/event.Card.jsx:1]: Dont user img tag. user `Image` component from next/image instead. 

```jsx
import Image from "next/image"
```

**Review by indrazm — CHANGES_REQUESTED** [review]
Almost done.

File: src/components/Event/component/event.Card.jsx
Comment (lutfinoorm) [reply src/components/Event/component/event.Card.jsx:1]: Okay, mas. Do I need to rewrite in the same branch? Or make a new branch

**Comment by lutfinoorm** [comment]
I've removed the div from the page.js and used \<Image> for the image in EventCard.jsx, Mas.

**Comment by sarasseres** [comment]
mas ini aman klo mau di merge kah ? mas @indrazm

**Pull request merged**